### DISCUSSION TITLE
Problem for update packages with composer

### DISCUSSION THREAD

**Discussion by karaka200** [symfony/symfony#42722] [Q&A] [answered]
I use `composer show -o`
and show this list:

```
composer/package-versions-deprecated 1.11.99.2   1.11.99.3
doctrine/dbal                        2.13.2      3.1.1
phpstan/phpstan                      0.12.95     0.12.96
psr/cache                            2.0.0       3.0.0
psr/container                        1.1.1       2.0.1
psr/link                             1.1.1       2.0.1
psr/log                              2.0.0       3.0.0
```

but when I use `composer update` show:

`Nothing to install, update or remove`

How can I update this packages?

**Comment by osavchenko** [comment, marked as answer]
Some other packages could block these packages on existing versions. You can check why you can't update them with [prohibits](https://getcomposer.org/doc/03-cli.md#prohibits-why-not-) composer CLI command

**Reply by ahundiak** [reply]
Still can't quite get over the notion that someone can post something and then immediately mark it as an answer.  Is it an answer?  Well technically sure.  Is it a correct or useful answer?  Absolutely not.  The dbal package listed for example is what is known in composer as a yellow upgrade.  You can't upgrade to it without breaking something.

**Reply by stof** [reply]
@ahundiak this was selected as an answer by the author of the *question*, not by the author of the answer: 
![image](https://user-images.githubusercontent.com/439401/130982836-a05f795c-9ef9-40d0-92bc-8e998bab302b.png)
I guess that @karaka200 considered that answer useful for them.

**Reply by chyshyya** [reply]
Look into composer.json config file.
For doctrine/dbal you could see line with something:
`"doctrine/dbal": "2.13",`
If you want to update package to 3.1.1 then change setting to:
`"doctrine/dbal": "^3.1",`
But previosly you must know what you are doing!
